models/curvefi/ethereum/curvefi_ethereum_schema.yml

```diff
@@ -0,0 +1,85 @@
+version: 2
+
+models:
+  - name: curvefi_ethereum_view_pools
+    meta:
+      blockchain: ethereum
+      project: curvefi
+      contributors: [agaperste, yulesa]
+    config:
+      tags: ["ethereum", "curvefi", "view_pools"]
```

Comment:
can you add 'view' as a tag? we're thinking through our tagging strategy, so there may be tweaks in another future PR, but tags will help us with job orchestration downstream for automation